scripts/ltr_msmarco-passage/append_pred.py

```diff
@@ -0,0 +1,68 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+'''Converts MSMARCO's tsv collection to Anserini jsonl files with field configurations.'''
```

Comment:
PEP 8 wants line breaks, right?

Can we have descriptions of what the scripts do for each of them?